Dockerfile

```diff
@@ -1,12 +1,13 @@
 ARG NODE_TAG
 FROM node:${NODE_TAG}
 
-RUN mkdir /pipe
-WORKDIR /pipe
-
+RUN corepack enable
 RUN apk add wget 
 RUN wget -P / https://bitbucket.org/bitbucketpipelines/bitbucket-pipes-toolkit-bash/raw/0.4.0/common.sh
 
+RUN mkdir /pipe
+WORKDIR /pipe
+
 COPY tsconfig.json ./
 COPY pack*.json ./
 RUN npm ci
```

Comment:
Not quite sure how corepack works here? If we plan on using `pnpm` or `yarn` is this being hardcoded as `npm` fine?

Reply:
I agree that there is some confusion here.
1. corepack basically enable all 3 package managers that comes with nodeJS. It gives us the flexibility to use any of them in our project.
2. The hardcoded as `npm` in this Dockerfile is to install the dependencies for the pipeline itself, not the serverless project. We do this because we run the whole pipeline via `ts-node`. Probably it's better if we actually bundle our pipeline code to javascript instead.